src/core/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CommonProps } from '../utils/props';
+import { useTheme } from '../utils/hooks/useTheme';
+import { Button } from '../Buttons/Button';
+import '@itwin/itwinui-css/css/breadcrumbs.css';
+
+type BreadcrumbItemOwnProps<E extends 'a' | 'button' | 'span'> = {
+  /**
+   * Control which element the breadcrumb item should use.
+   *
+   * Use 'button' if you specify an `onClick` handler.
+   * Use 'a' if you also specify a meaningful `href`.
+   * Use 'span' if the item should not be clickable (e.g. for the current item).
+   * @default 'button'
+   */
+  element?: E;
```

Comment:
I also wonder if we leave option to set the element to the user, wont we get hybrid approaches which might not look good? like button + anchor... Maybe we should ask element/type on the `Breadcrumbs` component instead...Not sure yet

Reply:
But current breakcrumb is inactive and is of different type e.g. span so we can't control that. User can also pass completely different component, not `BreadcrumbItem`, we can't control everything.